catalog/pipeline/rhtap-service-push/0.1/rhtap-service-push.yaml

```diff
@@ -257,6 +257,24 @@ spec:
       runAfter:
         - push-sbom-to-pyxis
   finally:
+    - name: slack-webhook-notification
+      taskRef:
+        resolver: "git"
+        params:
+          - name: url
+            value: https://github.com/redhat-appstudio/release-service-bundles.git
+          - name: revision
+            value: main
+          - name: pathInRepo
+            value: catalog/task/slack-webhook-notification/0.1/slack-webhook-notification.yaml
+      params:
```

Comment:
According to this documentation: https://github.com/redhat-appstudio/release-service-bundles/blob/main/catalog/task/slack-webhook-notification/0.1/README.md

There is `message` and optionally `secretName` and `extraDataJsonPath`, but no `key-name` parameter.

Reply:
fixed